src/ray/gcs/gcs_server/gcs_server.h

```diff
@@ -34,6 +34,7 @@ struct GcsServerConfig {
   uint16_t redis_port = 6379;
   bool retry_redis = true;
   bool is_test = false;
+  std::string node_ip_address = "127.0.0.1";
```

Comment:
If the user sets node_ip_address to 127.0.0.1, it does not work? Maybe the default value can be empty.

Reply:
Good point. Will fix.